src/app/accounts/add-earn.ts

```diff
@@ -0,0 +1,68 @@
+import { getUser } from "@app/users"
+import { getBalanceForWalletId, intraledgerPaymentSendWalletId } from "@app/wallets"
+import { onboardingEarn } from "@config/app"
+import {
+  RewardInsufficientBalanceError,
+  RewardMissingMetadataError,
+  RewardNonValidTypeError,
+  ValidationError,
+} from "@domain/errors"
+import { getFunderWalletId } from "@services/ledger/accounts"
+import { RewardsRepository } from "@services/mongoose/rewards"
+import { getAccount } from "."
```

Comment:
we should avoid this. importing index could lead to future circular references. I think is better move getAccount to its own file.

Reply:
make sense, just reusing a pattern that was already here. how would you name that file?

Reply:
I'll do that on another PR

Reply:
just `get-account.ts`